### PR TITLE
fixed fullscreen key f11

### PR DESCRIPTION
**Pull request by berkeyvx**
This commit fix issue #68

This is my first contribute, please let me know if something is wrong

### CODE REVIEW

**Comment by eneshecan** [comment]
Hi, thanks for the help! We've gone into some redesigning which consists of changing the header menubar to a popover. First of all, we needed to remove the fullscreen accelerator from the menuitem because it's not getting focus on fullscreen due to the visibility of headerbar. Since we were one accelerator down, I thought let's remove all and move the needed accelerators into the code so that we can have a popover which accelerators kept us from using it.

So, I left the bug for you to fix. The things you need to do are;
* Rebase your branch
* Handle F11 key only on key *press* event as you did. I think we don't need to do anything on the key *release* event.
* Add `F11` to the fullscreen button in the ui.

**Comment by berkeyvx** [comment]
> Hi, thanks for the help! We've gone into some redesigning which consists of changing the header menubar to a popover. First of all, I needed to remove the fullscreen accelerator from the menuitem because it's not getting focus on fullscreen due to the visibility of headerbar. Since we were one accelerator down, I thought let's remove all and move the needed accelerators into the code so that we can have a popover which accelerators kept us from using it.
> 
> So, I left the bug for you to fix. The things you need to do are;
> 
> * Rebase your branch
> * Handle F11 key only on key _press_ event as you did. I think we don't need to do anything on the key _release_ event.
> * Add an `F11` text to the fullscreen menu button in the ui like `Fullscreen F11`.

I did not understand last task. Did you want me to modify ui/Mainwindow.ui file ?

**Comment by eneshecan** [comment]
Yes, please. Just make the `fullscreen_button` label `Fullscreen F11` for users to understand since this doesn't come automatically anymore.  Also, check if this looks nice on gui, if not please increase the number of spaces between two words.

**Comment by berkeyvx** [comment]
I did the changes. Can you check the commit again ? Also i could not try label that i have changed. Because i couldn't build

I am getting the following error while trying to build. 

```
(whatsapp-for-linux:19316): gtkmm-CRITICAL **: 19:22:17.799: gtkmm: object `close_to_tray_switch' not found in GtkBuilder file.

(whatsapp-for-linux:19316): gtkmm-CRITICAL **: 19:22:17.799: gtkmm: Gtk::Builder: widget `close_to_tray_switch' was not found in the GtkBuilder file, or the specified part of it.

** (whatsapp-for-linux:19316): CRITICAL **: 19:22:17.799: Gtk::Builder::get_widget(): dynamic_cast<> failed.
Segmentation fault (core dumped)

```

That happened after rebase.

**Comment by eneshecan** [comment]
Because you need to recompile resources. Try cleaning and building again

File: src/MainWindow.hpp [hunk 1]
```diff
@@ -18,6 +18,7 @@ class MainWindow
         bool on_key_release_event(GdkEventKey* keyEvent) override;
         bool on_window_state_event(GdkEventWindowState* windowStateEvent) override;
         bool on_delete_event(GdkEventAny* any_event) override;
+        bool on_key_press_event(GdkEventKey* event) override;
```
Comment (eneshecan) [inline src/MainWindow.hpp:21]: Could you please move this before `on_key_release_event` to have consistency?

File: src/MainWindow.cpp [hunk 2]
```diff
@@ -95,6 +95,18 @@ bool MainWindow::on_key_release_event(GdkEventKey* keyEvent)
     }
 }
 
+bool MainWindow::on_key_press_event(GdkEventKey* keyEvent){
```
Comment (eneshecan) [inline src/MainWindow.cpp:98]: Please relocate this also, once the declaration place is changed.

File: resource/ui/MainWindow.ui [hunk 3]
```diff
@@ -120,7 +120,7 @@
         </child>
         <child>
           <object class="GtkButton" id="fullscreen_button">
-            <property name="label" translatable="yes">Fullscreen</property>
+            <property name="label" translatable="yes">Fullscreen F11</property>
```
Comment (eneshecan) [inline resource/ui/MainWindow.ui:123]: Please add more spaces, i.e. 8 and check if looks nice.

**Comment by eneshecan** [comment]
Other than those, it looks nice!

**Comment by berkeyvx** [comment]
I forgot the delete generated files in src folder my bad. I aligned "Fullscreen" text with Close To Tray. I think it looks neat. All is done. Thanks for guidance

**Comment by eneshecan** [comment]
My pleasure. Thanks for the help :)

**Pull request merged**